### ISSUE TITLE
ERROR in unable to locate '/home/$USER/code/FelizPOC/public' at '/home/$USER/code/FelizPOC/public/**/*'

### ISSUE THREAD

**Issue by AlbertoDePena** [fable-compiler/webpack-config-template#18]
Seems like the `public` directory cannot be empty or else `copy-webpack-plugin` throws this error:

```batch
ERROR in unable to locate '/home/$USER/code/FelizPOC/public' at '/home/$USER/code/FelizPOC/public/**/*'
```

Not sure if this is an issue worth fixing... Just wanted to point it out =)

**Comment by alfonsogarciacaro** [comment]
Hi @AlbertoDePena, thanks for reporting! Hmm, didn't know about that. The `public` dir was empty in your case because you didn't have any assets? It's difficult to cover all use cases in the template, but maybe we could allow setting `assetsDir` to null and disabling the copy-webpack-plugin in that case. Although I'm not sure if this can cause issues with Webpack's dev server too 🤔

**Comment by AlbertoDePena** [comment]
@alfonsogarciacaro, yeah the public directory was empty because I didn't have any assets. Again, this might not be worth pursuing... Just wanted this mentioned so if someone else sees that issue they know the root cause. BTW, you guys are awesome! Fable, Feliz, just the general work of using FSharp for full stack is just amazing. Thank you so much for your work!!!
